test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/009_ATF_P_TC_navi_app_PTU_retry_sequence_finish_ForceProtectedService_ON_StartService_encrypted_true.lua

```diff
@@ -0,0 +1,244 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] PolicyTableUpdate is failed by any reason and "ForceProtectedService"=ON at .ini file
+-- [PTU] [GENIVI] SDL must start PTU for navi app right after app successfully registration
+--
+-- Description:
+-- In case SDL starts PolicyTableUpdate in case of no "certificate" at "module_config" section at LocalPT
+-- and PolicyTableUpdate is failed by any reason even after retry strategy (please see related req-s HTTP flow and Proprietary flow)
+-- and "ForceProtectedService" is ON(0x07, 0x0A, 0x0B, 0x0F) at .ini file
+-- and app sends StartService (<any_serviceType>, encypted=true) to SDL
+-- SDL must respond StartService (NACK) to this mobile app
+--
+-- 1. Used preconditions:
+-- RPC SetAudioStreamingIndicator is allowed by policy
+-- Navi app exists in LP, no certificate in module_config
+--
+-- 2. Performed steps
+-- 2.1 Register and activate navi application.
+-- 2.2 Wait PTU retry sequence to elapse.
+-- 2.3 Send Audio service.
+-- 2.4 Send Video service.
+-- 2.5 Send RPC service.
+-- 2.6 Send Hybrid service.
+--
+-- Expected result:
+-- 1. SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- 2. SDL invalidates PTU
+-- 3. SDL should return StartServiceNACK to Audio
+-- 4. SDL should return StartServiceNACK to Video
+-- 5. SDL should return StartServiceNACK to RPC
+-- 6. SDL should return StartServiceNACK to Hybrid
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"NAVIGATION"}
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyCeritificates = require('user_modules/shared_testcases/testCasesForPolicyCeritificates')
+local events = require('events')
+local Event = events.Event
+
+--[[ Local variables ]]
+local time_wait = (60 + 61 + 62 + 62 + 62 + 62)*1000
```

Comment:
@istoimenova, Thank you for update regarding _seconds_between_retries_ parameter